Podfile

```diff
@@ -32,4 +32,12 @@ use_frameworks!
     # Pods for testing
   end
 
+post_install do |installer|
```

Comment:
what's this?

Reply:
We were having a lot of deployment mismatches in our pods.. I think this is supposed to update our pods accordingly?